internal/logging/logging.go

```diff
@@ -55,3 +57,15 @@ func NewLogWriter(logDir, logFile string) *os.File {
 func WithAddress(c zerolog.Context, address *types.Address) zerolog.Context {
 	return c.Str("address", address.String()[0:8])
 }
+
+func WithAddressAttribute(a types.Address) slog.Attr {
+	return slog.String("address", a.String()[0:8])
+}
+
+func WithChannelIdAttribute(c types.Destination) slog.Attr {
```

Comment:
These helpers just help us be consistent in the key names we use for channel ids, objective ids and other common field; this should make parsing logs easier.